README.md

```diff
@@ -95,22 +117,6 @@ Navigate to the below link for API documentation and API playground
 
 http://localhost:8080/swagger-ui/index.html
 
-## CI/CD [Azure Pipeline]
-
-### Continuous Integration (CI)
-
-Currently CI is not implemented as part of Azure pipeline. Test automated integration will be taken up in next
-development cycle.
-
-### Continuous Deployment (CD)
-
-Check-in to the master branch triggers the Azure pipeline [auth-n-z] CI/CD process which will build the source code,
-generate the package and create the docker image. Docker image will then be deployed to Azure app services.
-
-Process Flow Diagram:
-
-![](src/main/resources/static/images/deployment-pipeline.png)
-
-Development API documentation link
+## Development API documentation link
 
 https://tdei-gateway.azurewebsites.net/swagger-ui/index.html
```

Comment:
Is this URL correct? When I click on it, I see nothing. I believe there is a **dev** missing in it.